### PR TITLE
[AIDAPP-47] Enhance the contact model for service management

### PR DESCRIPTION
**Pull request by ankit-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-47

### Technical Description

> Describe your changes in detail. This should include technical/architectural decisions and reasoning, if applicable.
> Removed the birthdate, high school grad year, other email field from contacts.
_______________________________________________

### CODE REVIEW

**Review by dgoetzit — COMMENTED** [review]
@ankit-canyon It looks like some tests are currently failing.

**Comment by ankit-canyon** [comment]
> @ankit-canyon It looks like some tests are currently failing.

HI @dgoetzit  

Please check now !

Thank you.

File: app-modules/contact/database/migrations/2024_04_15_162853_drop_columns_in_contacts_table.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        if (Schema::hasTable('contacts')) {
+            Schema::table('contacts', function (Blueprint $table) {
+                $table->dropColumn(['birthdate', 'hsgrad', 'email_2']);
+            });
+        }
+    }
+
+    public function down(): void
+    {
+        Schema::table('contacts', function (Blueprint $table) {});
```
Comment (dgoetzit) [inline app-modules/contact/database/migrations/2024_04_15_162853_drop_columns_in_contacts_table.php:53]: The down method should re-add the columns you've removed.

**Review by dgoetzit — CHANGES_REQUESTED** [review]

File: app-modules/contact/database/migrations/2024_04_15_162853_drop_columns_in_contacts_table.php (hunk 1 above)
Comment (ankit-canyon) [reply app-modules/contact/database/migrations/2024_04_15_162853_drop_columns_in_contacts_table.php]: > The down method should re-add the columns you've removed.

Hi,

Suggested changes are done !

Thank you.

**Review by dgoetzit — DISMISSED** [review]

**Review by dgoetzit — APPROVED** [review]

**Review by dgoetzit — APPROVED** [review]

**Review by dgoetzit — DISMISSED** [review]

**Review by dgoetzit — DISMISSED** [review]

**Review by joelicatajr — APPROVED** [review]
Changes approved.

**Review by joelicatajr — APPROVED** [review]

**Review by danharrin — DISMISSED** [review]

**Review by dgoetzit — APPROVED** [review]

**Pull request merged**